src/cmd/run.js

```diff
@@ -322,6 +322,8 @@ export class ExtensionRunner {
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
```

Comment:
Do I need to pass only the parameter into` ExtensionRunner.run()`, like this
`run(profile: FirefoxProfile, jsconsole): Promise<FirefoxProcess> {`